package.json

```diff
@@ -4,15 +4,18 @@
   "private": true,
   "dependencies": {
     "3box": "^1.19.1",
+
     "@1hive/1hive-ui": "^1.0.14",
     "@1hive/connect-agreement": "^0.9.5",
     "@1hive/connect-gardens": "^0.1.12",
     "@1hive/connect-react": "^0.9.8",
     "@1hive/evmcrispr": "^0.3.2",
+    "@aragon/ui": "^1.9.3",
```

Comment:
Note that we are using `@1hive/1hive-ui` package not the  `@aragon/ui` one. Let´s remove this

Reply:
Without it the `"sync-assets": "copy-aragon-ui-assets ./public",` copy-aragon-ui-assets failed. Need install it globally?

Reply:
Removed from package.json. Very weird now it works.

Reply:
Actually the 1hive UI tollkit already provides this bin file since it´s a fork of arragon/ui